### ISSUE TITLE
Need update for example

### ISSUE THREAD

**Issue by arvylagunamayor** [Roxeez/Moonlight.Example#1]
it seems you have made some changes on Moonlight, this example showing some errors.

**Comment by Roxeez** [comment]
Yes i'm gonna make a new example asap since lot of change has been made to Moonlight.

**Comment by LegendEvent** [comment]
when? :D Im trying to learn and this would be helpful
